pkgs/by-name/co/code-cursor/package.nix

```diff
@@ -98,12 +98,7 @@ in
     nativeBuildInputs =
       (oldAttrs.nativeBuildInputs or [ ]) ++ lib.optionals hostPlatform.isDarwin [ undmg ];
 
-    preInstall =
-      (oldAttrs.preInstall or "")
-      + lib.optionalString hostPlatform.isLinux ''
-        mkdir -p bin
-        ln -s ../cursor bin/cursor
-      '';
+    preInstall = (oldAttrs.preInstall or "");
```

Comment:
This is no longer necessary.  The entire line can be removed as you are not really changing preInstall.


```suggestion
```

Reply:
I left it in there purely out of caution, will remove it.